sdk/core/core-util/test/public/bytesEncoding.spec.ts

```diff
@@ -113,4 +101,35 @@ describe("Base64", function () {
       }
     });
   });
+
+  describe("bytes to hex", function () {
+    it("encodes a byte", function () {
+      const bytes = new Uint8Array(256).map((_, index) => index);
+      bytes.forEach((_, index) => {
+        const hex = uint8ArrayToString(bytes.slice(index, index + 1), "hex");
+        assert.equal(hex.length, 2, "Unexpected length for hex value.");
+        assert.equal(parseInt(hex, 16), index, "Unexpected hex value.");
+      });
+    });
+
+    it("encodes bytes", function () {
+      const bytes = new Uint8Array([97, 122, 117, 114, 101]); // 'azure' in utf8
+      const hex = uint8ArrayToString(bytes, "hex");
+      assert.equal(hex, "617a757265", "Unexpected hex value.");
+    });
+  });
+
+  describe("hex to bytes", function () {
+    it("decodes bytes", function () {
+      const hex = "617a757265";
+      const bytes = stringToUint8Array(hex, "hex");
+
+      assert.equal(bytes.length, 5);
```

Comment:
minor suggestion you could use `assert.sameOrderedMembers` to do the comparison of the bytes array I think: https://www.chaijs.com/api/assert/#method_sameorderedmembers

```ts
assert.sameOrderedMemembers([...bytes], [97, 112, 117, 114, 101]);
```